services/src/main/java/org/exoplatform/addons/gamification/service/mapper/EntityMapper.java

```diff
@@ -66,6 +68,9 @@ public static RuleEntity toEntity(Challenge challenge) {
     }
     challengeEntity.setAudience(challenge.getAudience());
     challengeEntity.setManagers(challenge.getManagers());
+    challengeEntity.setScore(20); // TODO: to be changed when add field score to DTO
+    challengeEntity.setArea("social"); // TODO: to be changed when add field domain to DTO
+    challengeEntity.setDomainEntity(DomainMapper.domainDTOToDomain(Utils.getDomainByTitle("social"))); // TODO: to be changed when add field domain to DTO
```

Comment:
check the last commit in feature/rewards please, and delete the default value of area and domain